loader/loader.c

```diff
@@ -5376,6 +5420,17 @@ VKAPI_ATTR VkResult VKAPI_CALL terminator_CreateInstance(const VkInstanceCreateI
     bool one_icd_successful = false;
 
     struct loader_instance *ptr_instance = (struct loader_instance *)*pInstance;
+    if (NULL == ptr_instance) {
+        loader_log(ptr_instance, VULKAN_LOADER_ERROR_BIT, 0,
+                   "terminator_CreateInstance: Instance pointer (NULL) passed down appears invalid. Possibly corrupted by active "
+                   "layer (Policy #LLP_LAYER_21)");
```

Comment:
This is the message returned when the instance pointer is null, not corrupt. "Appears invalid" could mean anything but here we know it is invalid, because it is null. Might be worth specifying that.